### PR TITLE
#98: Focus README on users.

### PR DESCRIPTION
**Pull request by redcatbear**
## Features
#98: Focus README on users

### CODE REVIEW

**Comment by kaklakariada** [comment]
The build is failing because sonar does not like `#` in branch names.

File: README.md [hunk 1]
```diff
@@ -29,157 +33,74 @@ Sonarcloud status:
...
+To use OpenFastTrace as a dependency in your [Gradle](https://gradle.org/) project:
 
 ```groovy
-repositories {
-    jcenter()
-}
 dependencies {
-    compile "org.itsallcode:openfasttrace:2.1.0"
+    compile "org.itsallcode:openfasttrace:2.2.0"
 }
```
Comment (kaklakariada) [inline README.md]: We should remove this section. Using oft as a library is not the typical use case.

File: README.md [hunk 2]
```diff
@@ -29,157 +33,74 @@ Sonarcloud status:
...
-    ```
-1. Merge to `master` branch
-1. Create a [release](https://github.com/itsallcode/openfasttrace/releases) of the `master` branch on GitHub.
-1. Sign in at [bintray.com](https://bintray.com)
-1. Go to the [Bintray project page](https://bintray.com/itsallcode/itsallcode/openfasttrace)
-1. There should be a notice saying "You have 6 unpublished item(s) for this package". Click the "Publish" link. Binaries will be available for download at [JCenter](https://jcenter.bintray.com/org/itsallcode/openfasttrace/)
-1. Publish to Maven Central by clicking the "Sync" button at https://bintray.com/itsallcode/itsallcode/openfasttrace#central. After some time the new version will appear at https://repo1.maven.org/maven2/org/itsallcode/openfasttrace/.
+If you want to learn how to build OpenFastTrace, please check our [Developer Guide](doc/developer_guide.md).
 
+You would like to contribute to OFT? Please check out our [Contributor Guide](../CONTRIBUTION.md) to get started.
```
Comment (kaklakariada) [inline README.md]:
```suggestion
You would like to contribute to OFT? Please check out our [Contributor Guide](CONTRIBUTION.md) to get started. 
```

File: doc/developer_guide.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# OpenFastTrace Developer Guide
+
+This document contains technical information for developers contributing to OpenFastTrace (short OFT).
+
+If you want to know more about how to contribute to OFT, please check out our [Contributor Guide](../CONTRIBUTION.md).
+
+## Build Time Dependencies
+
+The list below show all build time dependencies in alphabetical order. Note that except the Maven build tool all required modules are downloaded automatically by Maven.
```
Comment (kaklakariada) [inline doc/developer_guide.md]:
```suggestion
The list below shows all build time dependencies in alphabetical order. Note that except the Maven build tool all required modules are downloaded automatically by Maven.
```

File: doc/developer_guide.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+| [JUnit](https://junit.org/junit5)                                            | Unit testing framework                                 | Eclipse Public License 1.0    |
+| [Mockito](http://site.mockito.org/)                                          | Mocking framework                                      | MIT License                   |
+| [JUnit5 System Extensions](https://github.com/itsallcode/junit5-system-extensions) | JUnit extension for testing `System.x` calls    | Eclipse Public License 2.0     |
+| [Pitest](http://pitest.org/)                                                 | Mutation testing                                       | Apache License 2.0            |
+
+## Preparations
+
+OpenFastTrace uses [Apache Maven](https://maven.apache.org) as technical project management tool that resolves and downloads the build-dependencies before building the packages.
+
+### Installation of Build Initial Dependencies on Linux
```
Comment (kaklakariada) [inline doc/developer_guide.md]:
```suggestion
### Installation of Initial Build Dependencies on Linux
```

Comment (redcatbear) [reply doc/developer_guide.md]: Fixed typo.

**Comment by redcatbear** [comment]
@kaklakariada, fixed your findings. Please check again.

**Pull request merged**